### PR TITLE
Batch sql inserts for gff3 script

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #1080

### CODE REVIEW

File: scripts/utils.py [hunk 1]
```diff
@@ -18,6 +18,9 @@
 import yaml
 import pysam
 
+import requests.packages.urllib3
+requests.packages.urllib3.disable_warnings()
```
Comment (macieksmuga) [inline scripts/utils.py]: Is this code related to the issue being solved by this PR?

Comment (dcolligan) [inline scripts/utils.py]: No, this is extra.  There was just a bunch of spam in the script output.

**Comment by macieksmuga** [comment]
+1 Nice! Thanks @dcolligan - the code looks great, passed all my tests and resulted in a decent speedup of a moderately sized dataset (all gencode v24 annotations on Chromosome 17): The old code finished in ~ 98 seconds, this version in ~ 65 seconds.

**Comment by jeromekelleher** [comment]
+1. Merging.
